j2cl/lit/src/tokens/shell-tokens.css

```
@@ -110,37 +114,78 @@ shell-skip-link > a:focus {
+  height: 36px;
+  display: block;
+}
+
+shell-header > [slot="brand"] .j2cl-brand-stack {
+  display: inline-flex;
+  flex-direction: column;
+  line-height: 1.1;
+}
+
+shell-header > [slot="brand"] .j2cl-brand-name {
+  font-family: "Space Grotesk", "Inter", -apple-system, BlinkMacSystemFont,
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)